aws_okta_processor/cli.py

```diff
@@ -0,0 +1,45 @@
+"""
+Usage:
+  aws-okta-processor [options] <command> [<args>...]
+
+Options:
+  -h, --help     Show this screen.
+  --version      Show version.
+
+Commands:
+  authenticate  used to authenticate into AWS using Okta
+  get-roles     used to get AWS roles
+
+Help:
+  For help using this tool, please reach out to our Slack channel:
+  https://godaddy-oss-slack.herokuapp.com/
+
+See 'aws-okta-processor <command> -h' for more information on a specific command.
+"""  # noqa
+
+
+from docopt import docopt
+
+from . import __version__ as VERSION
+
+from . import commands
+
+
+def main():
+    """Main CLI entrypoint."""
+    args = docopt(__doc__, version=VERSION, options_first=True)
+
+    try:
+        argv = [args['<command>']] + args['<args>']
+        if args['<command>'] == 'authenticate':
+            options = docopt(commands.authenticate.__doc__, argv=argv)
+            command = commands.authenticate.Authenticate(options)
+            command.run()
+        elif args['<command>'] == 'get-roles':
+            options = docopt(commands.get_roles.__doc__, argv=argv)
+            command = commands.get_roles.GetRolesCommand(options)
```

Comment:
Nit: I wish this used the same naming pattern as `Authenticate`, which doesn't include the suffix `Command`. In this case `commands.get_roles.GetRoles` would match the pattern

Reply:
Why is there a mix of snake_case and PascalCase naming?

Reply:
I prefer a class to be a noun.

Reply:
Then Authenticate is the odd person out :-P

Reply:
Ah, I misunderstood the code

Reply:
Ya. I can change that.

Reply:
Obviously that is a breaking change.

Reply:
Renamed `GetRolesCommand` to `GetRoles` to avoid breaking change.